.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+on: [push]
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.13.x]
+        platform: [ubuntu-18.04, macOS-10.14, windows-2019]
```

Comment:
windows-2019 is Windwos Server 2019. I am unable to find an actual list of platforms. But maybe it would be better to test on Windows 10 instead?

Reply:
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idruns-on